api_docs/markdowns/api/bql/bql-aggs-query.md

```diff
@@ -24,21 +24,21 @@ A group-by is defined by:
   - ranges that define buckets for the group-by operation.
 
 #### Distinct GroupBy
-Only [[aggregables fields;analysis-datamodel?filter=agg:categorical]] can be used for distinct group-by operations. They are specified either by field name or by field, result size and sort order. By default, at most 100 results are returned sorted by value.
-The order key can be by `value` or `count`. The size is capped at `100`.
+Only [[aggregables fields;analysis-datamodel?filter=agg:categorical]] can be used for distinct group-by operations. They are specified either by field name or by field, result size and sort order. By default, at lease 1000 results are returned, sorted by value.
```

Comment:
"At least"?

Reply:
Yup: We now try to honor an explicit `size` parameter instead of having a hard default limit.